app/services/address_exporter.rb

```diff
@@ -35,11 +34,14 @@ def self.se_mailing_csv_str(address)
 
 
 
-  # if country == 'Sverige' return what the Swedish Postal service wants to see for a mailing address
+  # if country == 'Sverige' || 'SE' || 'Sveriges' standardize to "Sverige"
   def self.country_postal(address)
```

Comment:
This comment had me wondering why it wouldn't standardise 'SE-Sweden' and 'SE-Sverige'. Probably just an example of how quickly comments and code start to diverge. I would rather see the comment removed, and (perhaps - but maybe it is overkill) have the `country_postal` method renamed to something like `standardised_country_postal`